torch_geometric/loader/link_neighbor_loader.py

```diff
@@ -79,16 +94,46 @@ def _create_label(self, edge_label_index, edge_label):
 
         edge_label = torch.cat([pos_edge_label, neg_edge_label], dim=0)
 
-        return edge_label_index, edge_label
+        return edge_label_index, edge_label, edge_label_time
+
+    def _modify_node_time(self, edge_label_index, edge_label_time):
+        """If `edge_label_time` is `None` return `self.node_time_dict`.
+        Else, for edges in a batch replace `src` and `dst`
+        node times by the min across all edge times."""
+        if edge_label_time is None:
```

Comment:
Let's just call this function in case the task is temporal?

Reply:
Done.